liquidity_pool/src/token.rs

```diff
@@ -0,0 +1,12 @@
+#![allow(unused)]
+use soroban_sdk::{Bytes, BytesN, Env};
+
+soroban_sdk::contractimport!(file = "../soroban_token_spec.wasm");
```

Comment:
Since the contract spec is imported here, there should be no need to import it in the dependencies.